packages/manager/modules/log-to-customer/src/live-tail/controller.js

```diff
@@ -25,20 +31,100 @@ export default class LogToCustomerCtrl {
       createAccount: `${this.dbaasLogsURL}/order`,
     };
 
+    this.kind = this.kindInitValue;
     this.hasAtLeastOneDataStream = false;
+    this.errorMessage = null;
 
-    this.kind = this.kindInitValue;
+    this.init();
+  }
 
-    this.LogToCustomer.getDataStreams()
-      .then(({ streams, streamCount }) => {
-        this.logAccountStreams = streams;
-        this.hasAtLeastOneDataStream = streamCount > 0;
+  init() {
+    this.resetErrorMessage();
+    this.loading = true;
+    this.streamSubscriptions = [];
+
+    this.$q
+      .all([this.getDataStreams(), this.getSubscribedStreams()])
+      .catch(({ data }) => {
+        this.errorMessage = data.message;
       })
       .finally(() => {
         this.loading = false;
       });
   }
 
+  resetErrorMessage() {
+    this.errorMessage = null;
+  }
+
+  deleteLogSubscription(subscription) {
+    this.resetErrorMessage();
+    // eslint-disable-next-line no-param-reassign
+    subscription.isLoading = true;
+
+    this.LogToCustomerService.delete(
+      `${this.logSubscriptionUrl}/${subscription.subscriptionId}`,
+    )
+      .then(({ data }) => {
+        this.LogToCustomerService.pollOperation(
+          subscription.serviceName,
+          data,
+        ).then(() => this.init());
+      })
+      .catch(({ data }) => {
+        this.errorMessage = data.message;
+      })
+      .finally(() => {
+        // eslint-disable-next-line no-param-reassign
+        subscription.isLoading = false;
+      });
+  }
+
+  getDataStreams() {
+    return this.LogToCustomerService.getDataStreams().then(
+      ({ streams, streamCount }) => {
+        this.logAccountStreams = streams;
+        this.hasAtLeastOneDataStream = streamCount > 0;
+      },
+    );
+  }
+
+  async getSubscribedStreams() {
```

Comment:
try to be consistent. You're using `async` and `await` pattern here but not in `getDataStreams`

Reply:
It's mainly because otherwise in this function I would have to chain three promises, which I find more difficult to read and follow, knowing that in addition the calls can be made in parallel. 
Using await allows me to me to make calls at the same time and make the code more readable. There is no such need in getDataStreams() function